### DISCUSSION TITLE
Tooltip for table cells with truncated data

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#6664] [Q&A]
We currently use Spectrum React for our web application.  My question is related to the table component.  On some pages we have tables with many columns.  And on smaller screens, the columns get resized and the cell data gets truncated.  In such a case, and only for the cells with truncated data, we'd like to allow the user to see the full text using a tooltip.  Our initial research into this has not produced any results and we have not found a way to do this.  But perhaps we are overlooking something.  Is it possible?

**Comment by reidbarber** [comment]
Try `overflowMode="wrap"`:  https://react-spectrum.adobe.com/react-spectrum/TableView.html#overflow-mode
